apps/ledger-live-desktop/src/renderer/screens/dashboard/index.tsx

```diff
@@ -168,3 +153,10 @@ const PortfolioGrid = styled(Grid).attrs(() => ({
 }))`
   grid-template-columns: 2fr 1fr;
 `;
+
+const SwapPortfolioGrid = styled(Grid).attrs(() => ({
+  columnGap: 2,
+  columns: 2,
+}))`
+  grid-template-columns: 2fr 1fr;
+`;
```

Comment:
Maybe you can refactor a little bit to combine `marketPerformanceEnabled` & `ptxSwapLiveAppOnPortfolio` ?